cmake/ramses/platformConfig.cmake

```diff
@@ -195,6 +195,12 @@ IF(${CMAKE_SYSTEM_NAME} MATCHES "Android")
     SET(ENV{PKG_CONFIG_SYSROOT_DIR} ${CMAKE_SYSROOT})
 ENDIF()
 
+IF((${CMAKE_SYSTEM_NAME} MATCHES "Darwin") OR (${CMAKE_SYSTEM_NAME} MATCHES "iOS"))
+    ADD_FLAGS(RAMSES_C_FLAGS "-x objective-c")
+    SET(CMAKE_XCODE_ATTRIBUTE_CLANG_CXX_LANGUAGE_STANDARD "c++20")
```

Comment:
the default build standard for ramses is c++14. Is there a reason/need to use 20 on xcode? If not, I'd suggest to use 14, otherwise please add a TODO (See other comments)

Reply:
There is no reason, so I updated it to be `c++14`. It is annoying that this line is necessary at all, since for some reason CMake does not communicate the language standart to Xcode correctly.